README.md

```diff
@@ -1 +1,213 @@
-# LeaderBoard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] Please remove all the comments and instructions after you're done setting up your README 👍

Reply:
The former reviewer asked that you remove these commented-out codes after setting up the README. Kindly implement the suggestion. Leaving commented-out codes makes your code hard to read, it makes the code look messy and it takes up space. 👍